Include/internal/pycore_frame.h

```diff
@@ -58,26 +58,21 @@ typedef struct _PyInterpreterFrame {
     PyObject *f_builtins; /* Borrowed reference. Only valid if not on C stack */
     PyObject *f_locals; /* Strong reference, may be NULL. Only valid if not on C stack */
     PyFrameObject *frame_obj; /* Strong reference, may be NULL. Only valid if not on C stack */
-    // NOTE: This is not necessarily the last instruction started in the given
-    // frame. Rather, it is the code unit *prior to* the *next* instruction. For
-    // example, it may be an inline CACHE entry, an instruction we just jumped
-    // over, or (in the case of a newly-created frame) a totally invalid value:
-    _Py_CODEUNIT *prev_instr;
+    /* The instruction that is currently executing (possibly not started yet). */
+    _Py_CODEUNIT *instr_ptr;
     int stacktop;  /* Offset of TOS from localsplus  */
-    /* The return_offset determines where a `RETURN` should go in the caller,
-     * relative to `prev_instr`.
-     * It is only meaningful to the callee,
-     * so it needs to be set in any CALL (to a Python function)
-     * or SEND (to a coroutine or generator).
-     * If there is no callee, then it is meaningless. */
-    uint16_t return_offset;
+    /* The next_instr_offset determines where the next instruction is relative
+     * to instr_ptr. It enables us to keep instr_ptr pointing to the current
+     * instruction until it is time to begin executing the next one. This is
+     * necessary for tracebacks and tracing. */
```

Comment:
Clarify what it is that's necessary for tracebacks and tracing (it's keeping `instr_ptr` pointing to the current instruction -- not having `next_instr_offset`.)
```suggestion
     * instruction until it is time to begin executing the next one, which is
     * necessary for tracebacks and tracing. */
```